notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -577,14 +589,16 @@
    "metadata": {},
    "source": [
     "In general\n",
-    "$\\mathbf{P}_{X\\tilde{T}} \\mathbf{P}_{\\tilde{T}X} = \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T \\mathbf{C}^{1/2}$ is not a symmetric matrix, and so it is not possible to define an orthonormal $\\mathbf{P}_{XT}$ such that $\\mathbf{P}_{TX}=\\mathbf{P}_{XT}^{T}$. \n",
+    "$\\mathbf{P}_{X\\tilde{T}} \\mathbf{P}_{\\tilde{T}X} = \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T \\mathbf{C}^{1/2}$ is not a symmetric matrix, and so it is not possible to define an $\\mathbf{P}_{XT}$ such that $\\mathbf{P}_{TX}=\\mathbf{P}_{XT}^{T}$. \n",
     "Consistently with the case of sample-space PCovR, we define \n",
     "\n",
     "\\begin{equation}\n",
     "\\begin{split}\n",
     "\\mathbf{P}_{XT} =& \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}\\hat{\\mathbf{\\Lambda}}_{\\mathbf{\\tilde{C}}}^{1/2} \\\\\n",
     "\\mathbf{P}_{TX} =& \\hat{\\mathbf{\\Lambda}}_{\\mathbf{\\tilde{C}}}^{-1/2} \\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T  \\mathbf{C}^{1/2} \\\\\n",
-    "\\mathbf{P}_{TY} =& \\hat{\\mathbf{\\Lambda}}_{\\mathbf{\\tilde{C}}}^{-1/2} \\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T  \\mathbf{C}^{-1/2} \\mathbf{X}^T \\mathbf{Y}, \\\\\n",
+    "\\mathbf{P}_{TY} =&(\\mathbf{T}^T\\mathbf{T})^{-1}\\mathbf{T}^T\\mathbf{Y} \\\\\n",
+    "           =&(  \\mathbf{P}^T_{XT}\\mathbf{X}^T\\mathbf{X} \\mathbf{P}_{XT})^{-1} \\mathbf{P}^T_{XT}\\mathbf{X}^T\\mathbf{Y} \\\\\n",
+    "           =&\\hat{\\mathbf{\\Lambda}}_{\\mathbf{\\tilde{C}}}^{-1/2} \\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T  \\mathbf{C}^{-1/2} \\mathbf{X}^T \\mathbf{Y} \\\\\n",
```

Comment:
I think this confuses rather than clarifies.

Reply:
Here I add the detailed derivation of P_TY. I think this will help readers understand how to get the formula for P_TY.

![image](https://user-images.githubusercontent.com/56018055/97762463-e88a5400-1b08-11eb-8a1a-7db6d3f67791.png)

Reply:
I understand that -- I just don't agree that it adds _clarity_ or is necessary. If you feel it is necessary, please delete the second line. I think it's reasonable to expect readers to be able to go from (T^T T)^-1 T^T Y to the PTY.

Reply:
Okay.